kolibri/core/assets/src/mixins/commonCoreStrings.js

```diff
@@ -365,6 +365,11 @@ export const coreStrings = createTranslator('CommonCoreStrings', {
     message: 'Search',
     context: 'Test used to indicate the Kolibri search field.',
   },
+  searchForUser: {
+    message: 'Search for a user...',
+    context:
+      'Text which appears in the search field above the table with users from whose to choose from (e.g. when enrolling learners to a class, selecting users to sync, etc.)',
```

Comment:
Text which appears in the search field above the table with users from ~~whose~~ whom to choose ~~from~~ (e.g. when enrolling learners to a class, selecting users to sync, etc.)

Reply:
Thanks, will do